src/probnum/randprocs/covfuncs/_covariance_linear_operator.py

```diff
@@ -50,21 +50,24 @@ def __init__(
         self,
         x0: np.ndarray,
         x1: Optional[np.ndarray],
-        shape,
-        evaluate_dense_matrix: callable,
-        keops_lazy_tensor: Optional["LazyTensor"] = None,
+        shape: Tuple[int],
```

Comment:
```suggestion
        shape: ShapeType,
```
where `ShapeType` is defined in `probnum.typing`.